docs/guides/cron.rst

```diff
@@ -0,0 +1,30 @@
+Cron
+====
+
+If you setup cron correctly as described in :doc:`/admin/cron` 
+special hooks will be triggered so you can register for these hooks from your own code.
+
+The example below registers a function for the daily cron.
+
+.. code:: php
+	
+	function my_plugin_init() {
+	    elgg_register_plugin_hook_handler('cron', 'daily', 'my_plugin_cron');
+	}
+	
+
+If timing is important in your cron hook be advised that the functions
+are executed in order of registration. This could mean that your function may
```

Comment:
Any use cases? You are not going to run a daily cron for a time sensitive operation - I imagine you would use a minute cron. And if cron takes more than a minute to run, then it probably is not a good idea to use cron for such an operation. You should at lease vroom to shutdown.